src/bundle/Resources/config/routing.yml

```diff
@@ -4,8 +4,17 @@ ezplatform.dashboard:
         _controller: 'FrameworkBundle:Template:template'
         template: 'EzPlatformAdminUiBundle:dashboard:dashboard.html.twig'
 
+_ezpublishLocation:
+    path: /content/location/{locationId}/{languageCode}/{viewType}/{layout}
+    defaults:
+        _controller: ez_content:viewLocation
+        viewType: full
+        layout: true
+        languageCode: 'eng-GB'
```

Comment:
You can't hardcode `eng-GB` here. It can be completely different depending on the setup.